consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/IndividualRequestConsumerTest.java

```diff
@@ -364,22 +298,24 @@ private IndividualResponseType generateResponse() {
      * Need to extend the consumer class under test for overriding base class methods
      * of the getClient as it is abstract method.
      */
-    private class IndRequestConsumer extends IndividualRequestConsumer {
+    protected class IndConsumerGateway extends IndividualRequestConsumer
+    {
         /**
          * Get the client for the specified target application. If the client is not cached already, a new client
          * connection is created otherwise the already cached client is returned.
          *
          * @param targetApplicationCode the target application code
-         * @param serviceType           the service type associated with the target application code
-         * @param webServiceEndPoint    the Web Service End Point URL
-         * @param connectionTimeOut     the connection time out value
-         * @param receiveTimeOut        the acknowledgement time out value
+         * @param serviceType the service type associated with the target application code
+         * @param webServiceEndPoint the Web Service End Point URL
+         * @param connectionTimeOut the connection time out value
+         * @param receiveTimeOut the acknowledgement time out value
          * @return the target application end point port bean i.e. the client interface.
          */
         @Override
-        public ITargetAppInternalEndpointPortType getClient(final String targetApplicationCode,
-                                                            final String serviceType, final String webServiceEndPoint,
-                                                            final long connectionTimeOut, final long receiveTimeOut) {
+        public ITargetAppInternalEndpointPortType getClient (final String targetApplicationCode,
+                                                             final String serviceType, final String webServiceEndPoint,
+                                                             final long connectionTimeOut, final long receiveTimeOut)
+        {

```

Comment:
Move "{" back to end of previous line?

Reply:
Please reserve cosmetic changes for new code - they shouldn't hold up approval.